sparktk-core/src/main/scala/org/trustedanalytics/sparktk/frame/internal/ops/join/JoinRddFunctions.scala

```diff
@@ -36,6 +37,40 @@ object JoinRddFunctions extends Serializable {
     new BroadcastJoinRddFunctions(joinParam)
 
   /**
+   * Perform cross join and return a FrameRdd with the cartesian product of the right and left frames.
+   *
+   * If the right frame has a column with the same name as the left frame, the result frame will have a column with the
+   * "_R" suffix added to the end of the column name with the right frame's column data.
+   *
+   * @param left Left frame for cross join
+   * @param right Right frame for cross join
+   * @return Result frame with the cartesian product of the left and right frames
+   */
+  def crossJoin(left: FrameRdd, right: FrameRdd): Frame = {
+    val leftDataFrame = left.toDataFrame
+    val rightDataFrame = right.toDataFrame
+
+    // Perform cross join using the data frames
+    val joinedFrame = leftDataFrame.join(rightDataFrame)
+
+    // Create the new schema by starting with the left frame's schema
+    var joinedSchema = left.frameSchema
+
+    // Add columns from the right frame to the new new schema, while checking for duplicate column names
```

Comment:
I think we already have logic to merge two schemas together...  look in Schema.scala